go.mod

```diff
@@ -3,16 +3,16 @@ module github.com/ONSdigital/dp-download-service
 go 1.13
 
 require (
-	github.com/ONSdigital/dp-api-clients-go v1.9.0
+	github.com/ONSdigital/dp-api-clients-go v1.11.1-0.20200610132403-b60b48e247ca
```

Comment:
Needs update to fixed version when PR is approved…
https://github.com/ONSdigital/dp-api-clients-go/pull/52#pullrequestreview-428953091

Reply:
updated to v1.12.0